Gems/PhysX/Code/Source/EditorColliderComponent.cpp

```diff
@@ -204,7 +173,7 @@ namespace PhysX
         if (auto serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
         {
             serializeContext->Class<EditorColliderComponent, EditorComponentBase>()
-                ->Version(1 + (1<<PX_PHYSICS_VERSION_MAJOR)) // Use PhysX version to trigger prefabs recompilation when switching between PhysX 4 and 5.
+                ->Version(2)
```

Comment:
@SergeyAMZN do we need to keep the previous logic? IIRC correct that was quite important if customers would switch between PhysX 4 or 5

Reply:
I talked to @SergeyAMZN and yes, we need to keep this, I thought it was only for Mesh Collider, but that's not true, it's needed for whenever it cooks data, which it's done when using cylinder (handled by this class) or non-uniform scale, I added it back.

Reply:
Okay perfect thanks 👍